src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3573,6 +3748,19 @@ public bool Reachable
                     return _knownNullState.Capacity > 0;
                 }
             }
+
+            public override string ToString()
```

Comment:
>ToString [](start = 35, length = 8)

Is this for debugging purposes? Consider adding a specialized helper for that rather than to changing ToString.

Reply:
I've changed that to use debugger display method instead, in follow-up PR https://github.com/dotnet/roslyn/pull/26828